modules/flyte/variables.tf

```diff
@@ -0,0 +1,49 @@
+variable "azurerm_kubernetes_cluster_oidc_issuer_url" {
+  type        = string
+  description = "AzureRM Kubernetes Cluster OIDC issuer url"
+}
+
+variable "azurerm_resource_group_location" {
```

Comment:
consider combining variables based on purpose. for example:
`azurerm_resource_group_location` and `azurerm_resource_group_name`
 
can be something like 

```hcl
variable "azurerm_resource_group" {
  description = "AzureRM Resource Group info"
  type = object({
    location = string
    name = string
  })
}
```